web/src/main/java/org/cbioportal/web/ResourceDataController.java

```diff
@@ -113,10 +113,10 @@
     @RequestMapping(value = "/studies/{studyId}/resource-data", method = RequestMethod.GET,
         produces = MediaType.APPLICATION_JSON_VALUE)
     @ApiOperation("Get all resource data for a study")
-    public ResponseEntity<List<ResourceData>> getAllClinicalDataInStudy(
+    public ResponseEntity<List<ResourceData>> getAllResourceDataInStudy(
```

Comment:
I think this is confusing because it could mean get all sample/patient/study resource data. Maybe `getAllStudyResourceDataInStudy`?

Reply:
how about `getAllResourceDataForStudy`?

Reply:
for consistency with the other methods how about `getAllResourceDataOfStudy`